src/kp2bw/bw_serve.py

```
@@ -0,0 +1,588 @@
+    ) -> None:
+        """Upload one attachment via multipart ``POST /attachment``."""
+        resp = await client.post(
+            "/attachment",
+            params={"itemid": item_id},
+            files={"file": (filename, data)},
+        )
+        if resp.status_code >= 400:
+            raise BitwardenClientError(
+                f"Attachment upload failed for {filename!r} on item {item_id}: "
+                f"HTTP {resp.status_code}"
+            )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)